oaknational/Oak-Web-Application
# test: make every download restricted - LESQ-1065

## Description

Music year: {owa_music_year}

-  Previously only the lesson downloads from `account-security-68rkee` were geo-restricted behind a featureflag. This PR makes all download pages geo-restricted if you have `"isRegionAuthorised": false` and are using the featureflag.

## How to test

1. Go to https://deploy-preview-2774--oak-web-application.netlify.thenational.academy
Go to any downloads page (feature flag enabled) if you have `"isRegionAuthorised": false` in clerk you should see the
`Sorry, downloads for this lesson are not available in your country` page

## Screenshots

https://github.com/user-attachments/assets/2dfdaefa-faa0-4a6c-b51c-ef6dea6c1d75



## Checklist
